### DISCUSSION TITLE
Why is "accept-ranges" set to none on playback server?

### DISCUSSION THREAD

**Discussion by yelodevopsi** [bluenviron/mediamtx#4028] [Questions]
### Question

See that we have problems "seeking" when using the playback URL to the mp4-files.
Is there a reason for not using "Accept-Ranges"? @aler9 
Anything with the segments etc?

Else I'll try to implement support for this, maybe on a new playback URL.

I'm looking at the `on_get.go`file line:27
https://github.com/bluenviron/mediamtx/blob/7191c85e3bca0ff3947ab22c138a1f6d18f68892/internal/playback/on_get.go#L27

**Comment by aler9** [comment]
Hello @yelodevopsi, the reason is that playback streams are dynamically generated during each request, therefore it's impossible to seek something that does not exist yet. A caching system might be useful but it's not trivial to implement since it has to take into account the fact that recording segments might get added, deleted or even changed (in case of segments that are not finalized yet).

**Reply by cooperbrown9** [reply]
I had this same question today, thank you for your answer
